robot/resources/lib/python_keywords/neofs_verbs.py

```diff
@@ -1,15 +1,17 @@
 import json
 import logging
 import os
+import random
 import re
 import uuid
 from typing import Any, Optional
 
 import allure
 import json_transformers
 from cluster import Cluster
-from common import ASSETS_DIR, TEST_OBJECTS_DIR, NEOFS_CLI_EXEC, WALLET_CONFIG
+from common import ASSETS_DIR, NEOFS_CLI_EXEC, TEST_OBJECTS_DIR, WALLET_CONFIG
```

Comment:
Not the most useful change.

Reply:
Linter upon commits, will remove the change.

Reply:
this is fixed